src/predefined-variables.ts

```diff
@@ -37,12 +43,16 @@ export function init ({gitData, argv}: PredefinedVariablesOpts): {[name: string]
         CI_SERVER_FQDN: `${gitData.remote.host}`,
         CI_SERVER_HOST: `${gitData.remote.host}`,
         CI_SERVER_PORT: `${gitData.remote.port}`,
-        CI_SERVER_URL: `https://${gitData.remote.host}:443`,
+        CI_SERVER_URL: CI_SERVER_URL,
         CI_SERVER_PROTOCOL: "https",
         CI_API_V4_URL: `https://${gitData.remote.host}/api/v4`,
```

Comment:
Please add `:${gitData.remote.port}` to support self-hosted

Reply:
this might be tricky, because currently, 

`gitData.remote.port` returns either `SERVER_PORT` OR the `SERVER_SHELL_SSH_PORT` 

depending on the output of the `git remote -v`

i think there's currently some bug, will create an mr to fix it before updating this mr